### PR TITLE
New ui developed

### PR DESCRIPTION
**Pull request by ksrajath7**
Created UI that displays all the attached images in a formatted grid layout.

### CODE REVIEW

File: screens/Editor.tsx [hunk 1]
```diff
@@ -68,16 +76,22 @@ export default function Editor({ route, navigation }: EditorProps) {
     let result  = await ImagePicker.launchImageLibraryAsync({
       mediaTypes: ImagePicker.MediaTypeOptions.Images,
       allowsEditing:true,
-      aspect:[4,3],
+      // aspect:[4,3]
     })
     if(!result.cancelled){     
       // dispatch(updateEntry({...entry, image: result.uri}))
-      dispatch(updateEntry({...entry, image: result.uri}))
+      dispatch(updateEntry({...entry, image:[...entry.image || [], result.uri] }))
```
Comment (naviji) [inline screens/Editor.tsx:83]: `...entry.image || []`
ethine artham entha?

File: screens/Editor.tsx [hunk 2]
```diff
@@ -94,25 +108,22 @@ export default function Editor({ route, navigation }: EditorProps) {
...
-          {/* <TouchableOpacity onPress={()=>{dispatch(removeEntry(entry.id))}}>
-          <Icons name={'delete'} size={30} color='red' style={{marginLeft:10}}/>
-
-          </TouchableOpacity> */}
         </View>
       </View>
+      
 }
+  <Text style={{fontSize: 20, fontWeight: "bold", marginHorizontal:20, marginTop:20}}>
+          Tell use about your day..
```
Comment (naviji) [inline screens/Editor.tsx:116]: typo : us instead of use

File: screens/Editor.tsx [hunk 3]
```diff
@@ -129,6 +140,8 @@ export default function Editor({ route, navigation }: EditorProps) {
             dispatch(updateEntry(updatedEntry));
           }}
           placeholder="Your note here"
+          autoFocus
+          onTouchStart={()=>{}}
```
Comment (naviji) [inline screens/Editor.tsx:144]: ethentha?

File: screens/Home.tsx [hunk 4]
```diff
@@ -221,23 +252,19 @@ export default function Home({ navigation }: HomeProps) {
...
 
   const { width, height } = Dimensions.get("window");
 
   return (
-    <View style={{ flex: 1, backgroundColor: "#ededed" }}>
-      <View style={{flex: 8}}>
+    <View style={{ flex: 1, paddingTop:80 }}>
+      <View style={{flex: 2, paddingBottom:20}}>
       <Carousel
+      layout="tinder"
```
Comment (naviji) [inline screens/Home.tsx:266]: tinder layout aakumbo today's card is at the bottom.
I think we should go back to the previous layout.

File: types.ts [hunk 5]
```diff
@@ -7,7 +7,7 @@ import { RouteProp } from '@react-navigation/native';
 export interface Entry {
     id: number,
     date: Date,
-    image?: string,
+    image?: Array<string>,
```
Comment (naviji) [inline types.ts:10]: image il enthokke undu. Array aakithu enthina?

**Comment by naviji** [comment]
Image view nannayi! :)

But eppozhum crop cheyyan chodikkunundu. 
Also large height ulla images attach cheytha image view break cheyyunnu.

File: types.ts (hunk 5 above)
Comment (ksrajath7) [reply types.ts:10]: Onniladhikam image save cheyyande

File: screens/Home.tsx (hunk 4 above)
Comment (ksrajath7) [reply screens/Home.tsx:266]: Ok

File: screens/Editor.tsx (hunk 1 above)
Comment (ksrajath7) [reply screens/Editor.tsx:83]: Angane or image enna array thodakkathil illallo.. appo updateentry vilikkana timeil ...entry.image enn mathram koduthal error kaanikunnu . Appo njan empty array koode koduthu.. ithu aah cs50 Il parayunnund. Ah contact listil section header add cheyyumbo

**Pull request closed without merging**